drizzlepac/haputils/catalog_utils.py

```diff
@@ -689,6 +702,11 @@ def verify_crthresh(self, n1_exposure_time):
                     self.reject_cats[cat_type] = True
                     log.info("{} catalog FAILED CR threshold.".format(cat_type))
 
+        # Ensure if any catalog is rejected, the remaining catalogs are also rejected
+        if any([True for k,v in self.reject_cats.items() if v == True]):
```

Comment:
This should have been:
```python
if any(self.reject_cats.values()):
```

Reply:
@mcara _Thank you Mihai!_  When I make the final version, I will change it to your suggestion.